credstash.py

```diff
@@ -581,6 +581,16 @@ def createDdbTable(region=None, table="credential-store", **kwargs):
         ]
     )
 
+    if tags:
+        tagset = []
+        for i in tags:
+            tag = i.split('=')
```

Comment:
Can you add some input validation here? 

We want to raise a friendly error if the tag input doesn't match the expected format.

Reply:
Good idea. Just saw we already have `key_value_pair()` which we are using for type validation with argparse. I'll make sure `--tags` is enforcing it also.

Reply:
Should be good now. Let me know if there is anything else :)